### ISSUE TITLE
Improve UI for LLM challenge and Submission

### ISSUE THREAD

**Issue by canyon289** [aivillage/llm_verification#19]
For example make overall modal window larger. Low priority and  gated by multiturn

![image](https://github.com/aivillage/llm_verification/assets/7213793/7808d33e-9555-414b-9ca9-df558bd98ba5)

**Comment by canyon289** [comment]
Now with multi turn
![image](https://github.com/aivillage/llm_verification/assets/7213793/2527334f-218a-4a4f-822e-7131b18243a2)
